pkg/manager/member/pd_member_manager.go

```diff
@@ -463,6 +463,12 @@ func (pmm *pdMemberManager) getNewPDServiceForTidbCluster(tc *v1alpha1.TidbClust
 			Selector: pdLabel,
 		},
 	}
+	svcSpec := tc.Spec.PD.Service
+	if svcSpec != nil {
+		pdService.Spec.ClusterIP = svcSpec.ClusterIP
+		pdService.Spec.LoadBalancerIP = svcSpec.LoadBalancerIP
+	}
```

Comment:
`.spec.pd.service.type` and `.spec.pd.service.annotations` should also be respected

Reply:
done